src/main/java/com/twilio/taskrouter/application/servlet/EventsServlet.java

```diff
@@ -25,48 +31,85 @@
 @Singleton
 public class EventsServlet extends HttpServlet {
 
-  public static final String LEAVE_MSG = "Sorry, All agents are busy. Please leave a message. "
+  protected static final String LEAVE_MSG = "Sorry, All agents are busy. Please leave a message. "
```

Comment:
A rule of thumb is a good practice to test the private API though the public API.

Your test is supposed to be decoupled from the **implementation details**. I think the test you write is good. But you don't have to change the public/protected API of your class just because you want shorten the assertion. It's totally ok to have an assertion like this:

```java
verify(twilioAppSettingsMock, times(1)).redirectToVoiceMail(
  "call_sid-content", "Sorry, All agents are busy. Please leave a message.");
```

It is supposed the test to drive the implementation, not backwards.